packages/client/src/app/game/views/character-picker/character-picker.component.ts

```diff
@@ -7,14 +7,14 @@ import { AVAILABLE_CHARACTERS, VIEWER } from '../../../shared/characters.const';
   styleUrls: ['./character-picker.component.scss']
 })
 export class CharacterPickerComponent implements OnInit {
-  @Input() private selectedCharacter: string = null;
+  @Input() selectedCharacter: string = null;
```

Comment:
ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/character-picker/character-picker.component.html (21,88   Property 'viewer' is private and only accessible within class 'CharacterPickerComponent'.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/character-picker/character-picker.component.html (4,18)  Property 'availableCharacters' is private and only accessible within class 'CharacterPickerComponent'.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/character-picker/character-picker.component.html (19,7)  Property 'selectedCharacter' is private and only accessible within class 'CharacterPickerComponent'.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/character-picker/character-picker.component.html (19,7)  Property 'viewer' is private and only accessible within class 'CharacterPickerComponent'.